docs/guides/cluster.md

```diff
@@ -69,3 +69,33 @@ with Diagram("Event Processing", show=False):
 ![event processing diagram](/img/event_processing_diagram.png)
 
 > There is no depth limit of nesting. Feel free to create nested clusters as deep as you want.
+
+## Clusters with icons in the label
```

Comment:
Node as Cluster functionality is not ready yet. It should be removed from this PR.